### DISCUSSION TITLE
https-portal Always Running

### DISCUSSION THREAD

**Discussion by ShaonPro** [outline/outline#7859] [Self Hosting] [answered]
This Container always trying to find something i am assuming. 
![image](https://github.com/user-attachments/assets/e389bc0c-4f5f-4b47-8694-1a01a7af7912)
![image](https://github.com/user-attachments/assets/eb96d41e-e51d-4e5e-9572-8f73a3a4fd8e)
![image](https://github.com/user-attachments/assets/1f794ee1-e706-4976-afe3-d8e65075ab53)

What is happening? how can i make this stable?

**Comment by tommoor** [comment, marked as answer]
The error is at the top of your screenshot here – 

<img width="548" alt="image" src="https://github.com/user-attachments/assets/6ba2d647-6567-42ac-8b43-dc0c674462e5">

It seems like LetsEncrypt is unable to access your server. This is more of a question for https://github.com/SteveLTN/https-portal than Outline, using this container is a suggested setup but is in no way required :)
